pkg/tables/impl/ethereum/client.go

```diff
@@ -137,6 +137,7 @@ func (c *Client) RunSQL(
 			From:      auth.From,
 			Nonce:     big.NewInt(0).SetInt64(nonce),
 			GasTipCap: gasTipCap,
+			GasLimit:  100000,
```

Comment:
@brunocalza I'm sure on this, but you might be able to use alchemy to do the gas estimate for you?
https://docs.alchemy.com/reference/eth-estimategas